command/admin.go

```diff
@@ -29,18 +29,17 @@ var AdminCmd = &cli.Command{
 
 var syncCmd = &cli.Command{
 	Name:   "sync",
-	Usage:  "Sync indexer with provider",
+	Usage:  "Sync indexer with provider. If no flags are provided returns a list of currently pending syncs.",
```

Comment:
Would it make sense to separate this into `sync list-pending` or something along those lines? Mixing it with `sync` such that it works without publisher ID _and_ does not sync instead returns list of pending syncs is something that I would find surprising as a user: There is no way I'd guess that what it would do.

Reply:
@masih done